src/main/java/core/basesyntax/db/FruitStorage.java

```diff
@@ -0,0 +1,9 @@
+package core.basesyntax.db;
+
+import core.basesyntax.model.Fruit;
+import java.util.ArrayList;
+import java.util.List;
+
+public class FruitStorage {
+    public static final List<Fruit> fruits = new ArrayList<>();
```

Comment:
make it Map<String, Int> and you will have O(1) complexsity to get fruit quantity